### ISSUE TITLE
Unable to update RNetCDF using `update.packages`

### ISSUE THREAD

**Issue by vlulla** [mjwoods/RNetCDF#129]
Trying to update my R install, on Debian Bookworm, I found that RNetCDF kept on failing to install with the error
```
... snipped ...
make[2]: Entering directory '/tmp/Rtmp9yvDRn/R.INSTALLdbc78d45d54/RNetCDF/src'
make[2]: /usr/lib/R/share/make/shlib.mk: No such file or directory
make[2]: *** No rule to make target '/usr/lib/R/share/make/shlib.mk'.  Stop.
make[2]: Leaving directory '/tmp/Rtmp9yvDRn/R.INSTALLdbc78d45d54/RNetCDF/src'
make[1]: *** [Makefile:14: all] Error 2
make[1]: Leaving directory '/tmp/Rtmp9yvDRn/R.INSTALLdbc78d45d54/RNetCDF/src'
ERROR: compilation failed for package ‘RNetCDF’
* removing ‘/usr/lib/R/site-library/RNetCDF’
* * restoring previous ‘/usr/lib/R/site-library/RNetCDF’
```

It appears that this error is raised from `tools/make-recursive.sh` script. I was surprised by this error message because most of the other R packages install/update without any issue. So, i got curious and tried to see what's wrong with `shlib.mk`.  And, to my complete surprise i found that `shlib.mk` is located at `/usr/share/R/share/make/shlib.mk` and not where the installation script is looking for it! I wonder if this is a problem on my end (however, i doubt it because this is a standard R install using Debian apt installation instructions on a GCP vm) or whether the installation script is somehow misconfigured.

I was able to install, by jiggering paths, RNetCDF for myself and sharing my findings in hopes that this can be helpful in making the installation/upgrade more robust. HTH!

* * * * * * * * * 
## System info

``` bash
~% cat /etc/os-release 
PRETTY_NAME="Debian GNU/Linux 12 (bookworm)"
NAME="Debian GNU/Linux"               
VERSION_ID="12"                              
VERSION="12 (bookworm)"
VERSION_CODENAME=bookworm
ID=debian                                                       
HOME_URL="https://www.debian.org/"                                                                                                               
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
~%
~%
~%
~% R --quiet --vanilla -e "sessionInfo()"
> sessionInfo()
R version 4.3.1 (2023-06-16)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Debian GNU/Linux 12 (bookworm)

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3 
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.21.so;  LAPACK version 3.11.0

locale:
 [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       
 [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   
 [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          
[10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   

time zone: Etc/UTC
tzcode source: system (glibc)

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_4.3.1
> 
> 
```

**Comment by mjwoods** [comment]
Thanks @vlulla , I aim to release a fixed package soon. I hope you can continue using the old package for now.

**Comment by mjwoods** [comment]
Resolved by #128
